web_readonly_bypass/__manifest__.py

```diff
@@ -1,33 +1,14 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#     This file is part of web_readonly_bypass,
-#     an Odoo module.
-#
-#     Copyright (c) 2015 ACSONE SA/NV (<http://acsone.eu>)
-#
-#     web_readonly_bypass is free software:
-#     you can redistribute it and/or modify it under the terms of the GNU
-#     Affero General Public License as published by the Free Software
-#     Foundation,either version 3 of the License, or (at your option) any
-#     later version.
-#
-#     web_readonly_bypass is distributed
-#     in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
-#     even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
-#     PURPOSE.  See the GNU Affero General Public License for more details.
-#
-#     You should have received a copy of the GNU Affero General Public License
-#     along with web_readonly_bypass.
-#     If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright (c) 2015 ACSONE SA/NV (<http://acsone.eu>)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
 {
     'name': 'Read Only ByPass',
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
     "author": "ACSONE SA/NV, Odoo Community Association (OCA)",
-    "maintainer": "ACSONE SA/NV,Odoo Community Association (OCA)",
+    "maintainer": "ACSONE SA/NV,Odoo Community Association (OCA),Kalpana Hemnani,Blutec Solutions,Kamil Muhammad",
```

Comment:
No, it should be only one author, and it the changes deserve to be here. The place for putting all the involved persons is in README.rst file, in the section contributors. There you can put several persons if they have worked on that.